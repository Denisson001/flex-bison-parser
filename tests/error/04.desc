Runtime error - index func: negative index