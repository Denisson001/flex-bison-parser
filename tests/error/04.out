Interpreter: runtime error - slice function: invalid index
